board/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+
+from user.models    import User
+from product.models import Product
+
+
+class Review(models.Model):
+    author           = models.ForeignKey(User, on_delete=models.SET_DEFAULT, default=1)
+    product          = models.ForeignKey(Product, on_delete=models.SET_DEFAULT, default=1)
+    title            = models.CharField(max_length=200)
+    contents         = models.TextField()
+    help_count       = models.IntegerField()
+    hit_count        = models.IntegerField()
+    created_at       = models.DateTimeField(auto_now_add=True)
+    updated_at       = models.DateField(auto_now=True)
+    review_image_url = models.URLField(max_length=2000)
+
+    class Meta:
+        db_table = 'reviews'
+
+class Question(models.Model):
+    author      = models.ForeignKey(User, on_delete=models.SET_DEFAULT, default=1)
+    product     = models.ForeignKey(Product, on_delete=models.SET_DEFAULT, default=1)
+    title       = models.CharField(max_length=200)
+    contents    = models.TextField()
+    created_at  = models.DateTimeField(auto_now_add=True)
+    updated_at  = models.DateField(auto_now=True)
+    is_private  = models.BooleanField(default=False)
+    has_comment = models.BooleanField(default=False)
```

Comment:
has_comment가 필요한 이유가 있나요? Question query set을 가져올 때, comment가 존재하는 Question만 조회해서 가져오면 될 것 같습니다~
```suggestion
    has_comment = models.BooleanField(default=False)
```

Reply:
네, 맞습니다.
그 방향으로 수정하겠습니다.

Reply:
Question (1) --- Comment(M) 관계로 설정한다는 말씀이신거죠?
has_comment column이 필요 없어 보입니다.

Reply:
조금 더 설명을 드리자면, has_comment column 없이 view에서 원하는 데이터를 출력할 수 있습니다.

Reply:
넵, Question (1) --- Comment(M) 관계로 설정하고자 합니다.
외래키 이용하여 조회하는 방향으로 코딩하도록 하겠습니다.